auth-web/tests/unit/services/business-lookup.service.spec.ts

```diff
@@ -14,15 +14,14 @@ describe('Business Lookup Services', () => {
     }
 
     const url = BusinessLookupServices.registriesSearchApiUrl + 'businesses/search/facets?start=0&rows=20' +
-      '&categories=legalType:A,BC,BEN,C,CC,CP,CUL,FI,GP,LL,LLC,LP,PA,S,SP,ULC,XCP,XL,XP,XS' +
-      `&query=value:${encodeURIComponent('FM1000002')}`
+      `&categories=legalType:SP&query=value:${encodeURIComponent('FM1000002')}`
 
     sinon.stub(axios, 'get').withArgs(url).returns(
       Promise.resolve({ data: { searchResults: { results: [result] } } })
     )
 
     // search and look at results
-    const results = await BusinessLookupServices.search('FM1000002')
```

Comment:
This was a syntax error because the second parameter was missing (and the third is optional).